sdk/src/confirmed_block.rs

```diff
@@ -0,0 +1,11 @@
+use crate::{
```

Comment:
For better or (probably) worse, client/src/rpc_request.rs contains a number of these RPC-only types for Rust RPC clients.    wdyt about moving this into there as well, purely so we keep them all in the same place until we do a bigger move

Reply:
I don’t love the idea of ledger depending on client, but as it doesn’t actually create a circular dependency, I suppose it’s okay until that bigger move...

Reply:
Yeah that's a little ugly.  Maybe that's good though, one day it'll motivate somebody to do the refactor :)